phac-nml/irida-next
# Workflow Execution: Show Page

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Adds in a basic Workflow Execution show page. Also updated another controllers turbo response as it bugged me.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/25466211/eee21920-0440-4368-b29c-a477c0ca3751)

![image](https://github.com/phac-nml/irida-next/assets/25466211/336a1c1a-f443-4fc0-ac1a-6304f40db050)

![image](https://github.com/phac-nml/irida-next/assets/25466211/6181236b-528d-43c7-9b42-e9219720734a)

**Workflow removed and redirected to index**

![image](https://github.com/phac-nml/irida-next/assets/25466211/41dcd10f-0f8b-46dc-a3b2-5d66371b458a)


## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Log in as `user1@email.com`
2. Navigate to `Workflow Exections` page
3. Click on the `ID` of a workflow in the table
4. See that you are on the show page and `Remove` it

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
